### ISSUE TITLE
https://wikimedia-gadgets.github.io/twinkle/Morebits.html error 404

### ISSUE THREAD

**Issue by NovemLinguae** [wikimedia-gadgets/twinkle#1668]
We have a link to some Morebits documentation in our readme.md file. It is currently returning an error 404. Is there a way to fix this, or should we remove the link?

https://wikimedia-gadgets.github.io/twinkle/Morebits.html

**Comment by NovemLinguae** [comment]
> This setup currently requires syncing all code changes to the gh-pages and then invoking the docs script. Not an optimal setup – ideally we should automate the whole thing (see #1405).

**Comment by fredster33** [comment]
I think that the docs folder could be moved to the master branch. GitHub Pages can serve from master; the docs URL would then be https://wikimedia-gadgets.github.io/twinkle/docs/Morebits.html

**Comment by fredster33** [comment]
And the docs script maybe could use GitHub Actions to automate it

**Comment by NovemLinguae** [comment]
Good idea merging the gh_pages branch into the master branch. Hiding stuff in other branches can cause it to fall off the radar. I saw this happen on another repo too.

If there's no objections, I'd accept a pull request that moved all that stuff to this branch.
